router/routes/courses.js

```diff
@@ -604,6 +605,10 @@ function  generateListings(data, user, cb) {
                         //         '</td>';
                         // }
                         // html += '</tr>';
+                        html +=
```

Comment:
Please explain this change

Reply:
Previous code only provides add/drop buttons. This part adds the feature of manage button.